env.sh

```diff
@@ -3,84 +3,83 @@
 # Environments' scripts.
 
 # OS
-if [[ $(uname) == 'Darwin' ]]; then
-    SYSTEM_NET="bridge"
-    SOURCE_DIR=$(dirname "$(realpath "$0")")
+if [[ "$(uname)" == 'Darwin' ]]; then
+    readonly SYSTEM_NET='bridge'
```

Comment:
[Readonly variables][1] in google shell-style guide.

[1]: https://google.github.io/styleguide/shell.xml?showone=Read-only_Variables#Read-only_Variables